spread/export_openstack_test.go

```diff
@@ -0,0 +1,89 @@
+package spread
+
+import (
+	"context"
+	"time"
+
+	gooseClient "github.com/go-goose/goose/v5/client"
```

Comment:
s/gooseClient/gooseclient/

Package names in Go are not typically cammel-cased.

Reply:
done